src/__internal__/fieldset/fieldset.style.js

```diff
@@ -1,17 +1,15 @@
 import styled, { css } from "styled-components";
 import PropTypes from "prop-types";
+import { margin } from "styled-system";
 import BaseTheme from "../../style/themes/base";
 
 const StyledFieldset = styled.fieldset`
-  ${({ ml, mb, theme }) => css`
+  ${() => css`
```

Comment:
You can just get rid of this function completely now that we're not using any of the parameters, i.e. 
```js
const StyledFieldset = styled.fieldset`
  ${margin}
  ..etc
`;
```

Reply:
and you can remove the `StyledFieldset.defaultProps` below as well because we don't need the theme anymore

Reply:
Hi Sam, good point thanks. Changed, but I think `defaultProps` must be defined because of `margin`.

Reply:
Good point you're right, I forgot about the margin bit!